x/launch/keeper/msg_server_trigger_launch_test.go

```diff
@@ -0,0 +1,113 @@
+package keeper
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func TestMsgTriggerLaunch(t *testing.T) {
+	k, _, srv, profileSrv, sdkCtx, _ := setupMsgServer(t)
+
+	ctx := sdk.WrapSDKContext(sdkCtx)
+	coordAddress := sample.AccAddress()
+	coordAddress2 := sample.AccAddress()
+	coordNoExist := sample.AccAddress()
+	chainIDNoExist, _ := sample.ChainID(0)
+
+	launchTime := types.DefaultMinLaunchTime
+	launchTimeTooLow := types.DefaultMinLaunchTime - 1
+	launchTimeTooHigh := types.DefaultMaxLaunchTime + 1
+
+	// Create coordinators
+	msgCreateCoordinator := sample.MsgCreateCoordinator(coordAddress)
+	_, err := profileSrv.CreateCoordinator(ctx, &msgCreateCoordinator)
+	if err != nil {
```

Comment:
```suggestion
	require.NoError
```

and same for others.